tests/unit/api/v1/test_status.py

```diff
@@ -38,3 +38,47 @@ def test_api_response_code(self):
         urlpath = "/api/v1/region/unexpected_region/status/day"
         code, resp = self.get(urlpath)
         self.assertEqual(404, code, urlpath)
+
+
+@mock.patch("ceagle.config.get_config")
+class HealthApiTestCase(test.TestCase):
+
+    def setUp(self):
+        super(HealthApiTestCase, self).setUp()
+        self.health_config = {"health": {
```

Comment:
It is better to pre-configure mock_get_config for every test method - this will really save LOC
https://docs.python.org/3.5/library/unittest.mock-examples.html#applying-the-same-patch-to-every-test-method